src/mcp/tools/auth/index.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint camelcase: 0 */
```

Comment:
Probably not for this PR, but it would be great if we could put this in .eslintrc.js instead of having this at the top of every file. Ideally, we'd exempt the mcp directory from the camlecase rule.

Reply:
Nevermind, fixed in this CL.